### ISSUE TITLE
DQA

### ISSUE THREAD

**Issue by sailor-ux** [wqb-uestc/DQA#3]
Thank you for your work on de-raining quality assessment!
I've been reading your code and there may be a problem:
In /DQA-master/DATA/IVIPC/DATAFolder.py , there is a line of code: "for target in sorted(os.listdir(dir)[:6]):", maybe it should be changed to: "for target in sorted(os.listdir(dir)):"
Without this change, neither the training set nor the test set will seem complete!
